sys/Makefile.dep

```diff
@@ -67,6 +67,10 @@ ifneq (,$(filter base64url,$(USEMODULE)))
   USEMODULE += base64
 endif
 
+ifneq (,$(filter netif,$(USEMODULE)))
+  USEMODULE += fmt
+endif
+
```

Comment:
This seems unrelated

Reply:
It popped out because netif `netif_addr_to_str` urtility uses `fmt` and the dependency is not explicitly made, should I split this commit?

Reply:
See https://github.com/RIOT-OS/RIOT/pull/14758#pullrequestreview-475476107 for a similar discussion.

Reply:
Thanks for the pointer, took a quick look your suggestion makes sense.